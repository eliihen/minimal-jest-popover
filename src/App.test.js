import React from 'react';
import ReactDOM from 'react-dom';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import App from './App';

Enzyme.configure({ adapter: new Adapter() });

jest.mock('random-id', () => {
  return jest.fn(() => 'STATIC_ID');
});

it('renders an unchanging snapshot', () => {
  const wrapper = Enzyme.mount(<App />);
  expect(wrapper).toMatchSnapshot();
});
