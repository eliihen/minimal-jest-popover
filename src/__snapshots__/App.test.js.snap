// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an unchanging snapshot 1`] = `
<App>
  <div
    className="App"
  >
    <header
      className="App-header"
    >
      <img
        alt="logo"
        className="App-logo"
        src="logo.svg"
      />
      <h1
        className="App-title"
      >
        Welcome to React
      </h1>
    </header>
    <Popover
      action="click"
      arrow={true}
      defaultIsOpen={false}
      modifiers={Object {}}
      motion={false}
      placement="auto"
    >
      <Manager
        className="manager"
        data-target-id="xsyzqqsddr"
        style={
          Object {
            "display": "inline",
          }
        }
        tag="div"
      >
        <div
          className="manager"
          data-target-id="xsyzqqsddr"
          style={
            Object {
              "display": "inline",
            }
          }
        >
          <TargetComponent
            action="click"
            closePopover={[Function]}
            id="xsyzqqsddr"
            openPopover={[Function]}
            tooglePopover={[Function]}
          >
            <Target>
              <div
                className="target-container"
              >
                <button>
                  Clicky!
                </button>
              </div>
            </Target>
          </TargetComponent>
        </div>
      </Manager>
    </Popover>
  </div>
</App>
`;
